components/masking_functions/include/masking_functions/charset_string_operations.hpp

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2018, 2019 Francisco Miguel Biete Banon. All rights reserved.
+   Copyright (c) 2023 Percona LLC and/or its affiliates. All rights reserved.
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; version 2 of the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software Foundation,
+   51 Franklin Street, Suite 500, Boston, MA 02110-1335 USA */
+
+#ifndef MASKING_FUNCTIONS_CHARSET_STRING_OPERATIONS_HPP
+#define MASKING_FUNCTIONS_CHARSET_STRING_OPERATIONS_HPP
+
+#include <cstddef>
+
+#include "masking_functions/charset_string_fwd.hpp"
+
+namespace masking_functions {
+
+charset_string repeat(const charset_string &str, std::size_t n);
```

Comment:
I'd prefer to leave `repeat()` as a visible function - at the next iteration of refactoring, when we extract `charset_string` into a common library, it will be easier to notice.
Added descriptions for the individual functions.

Reply:
OK. Sorry my comment about making repeat() static was bogus. I still not quite accustomed to github as a review tool.